### DISCUSSION TITLE
Problem about using vm to create bentoml service

### DISCUSSION THREAD

**Discussion by DanielLeung222** [bentoml/BentoML#2870] [Q&A] [answered]
Hi, when I use the vm to run the "bentoml serve service.py:svc --reload", there is no error message, but when I try to access to the given url of local host 127.0.0.1:3000, it shows nothing to me, can anyone help me? Many thanks!
![image](https://user-images.githubusercontent.com/108645351/183048508-1d7c282e-47fd-42cc-a469-cc4d339e06f8.png)
![image](https://user-images.githubusercontent.com/108645351/183049289-2b88eac1-95fe-4ed2-8197-26be9fe0bb56.png)
![image](https://user-images.githubusercontent.com/108645351/183050192-88d1a23e-fb33-41ad-9711-119cb844dbf3.png)

**Comment by aarnphm** [comment, marked as answer]
Hi @DanielLeung222, Can you check that your VM exposes 3000? It might be the case where the port is not exposed, and your local machine can't access it.
